fyle_accounting_mappings/views.py

```diff
@@ -279,7 +278,7 @@ def get_queryset(self):
         }
 
         if ((source_type == 'PROJECT' and destination_type == 'CUSTOMER') or\
-            (source_type == 'CATEGORY')) and app_name != 'XERO':
+            (source_type == 'CATEGORY')):
             filters['active'] = True
```

Comment:
bump library version as well, increment the patch version

Reply:
in the setup.py to 1.26.2 right?

Reply:
yes